scenario02/base-setup.sh

```diff
@@ -18,8 +18,8 @@
 appName="travelocity.com"
 tomcatHost=$tomcatHost
 tomcatPort=$tomcatPort
-tomcatUsername=scriptuser
-tomcatPassword=scriptuser
+tomcatUsername=$scriptuser
+tomcatPassword=$scriptuser
```

Comment:
Please define two different variable names for the tomcat username and tomcat password. It will be required when the username and the password of the tomcat server is different

Reply:
Fixed